docs/analysis/index.rst

```diff
@@ -13,9 +13,27 @@ Setting ``enable_monitoring: true`` in the configuration file will deploy a moni
 All these services are accessible on their default ports.
 For instance you'll be able to access grafana dashboards on port ``3000`` of the node hosting grafana.
 
-Some dashboards are available `here <https://github.com/BeyondTheClouds/kolla-g5k-results/tree/master/files/grafana>`_
+Some dashboards are available `here <https://github.com/BeyondTheClouds/kolla-g5k-results/tree/master/files/grafana>`_.
 
 Post-mortem
 -----------
 
 TODO
+
+Annotations
+-----------
+
+Enos embeds an Ansible plugin to add annotations in Grafana.
+These annotations are marked points which provide rich information about events
+when hovered over. To enable this option, the plugin must be loaded and the
+monitoring stack must be deployed (i.e. ``enable_monitoring: true`` should be
```

Comment:
I'm not sure if `enable_monitoring: true` should be in the configuration file (see comment below). But I may be wrong about how callbacks handle their variables.